provider/virtual_network/tftpboot.py

```
@@ -27,7 +27,32 @@
+        logging.debug("Create pxelinux.cfg without kickstart.")
+
+    contents = ["# pxelinux", f"default {entries[0]}"]
+    for entry in entries:
+        contents.append(f"label {entry}")
+        contents.append(f"kernel {entry}/kernel.img")
+        contents.append(f"initrd {entry}/initrd.img")
+        contents.append(kernel_cmdline)
+    return "\n".join(contents)
+
+
+def create_tftp_content(install_tree_url, kickstart_url, arch, entries=["linux"]):
```

Ruff (0.13.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)